src/components/NewCommentForm.tsx

```diff
@@ -14,24 +83,30 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': isErrorName })}
```

Comment:
It's great that you are using the `classnames` library for defining classes conditionally. This makes your code cleaner and easier to read.